c2corg_api/models/association.py

```diff
@@ -163,6 +163,7 @@ def _limit_waypoint_fields(query):
 def get_linked_waypoint_parents(document):
     return _limit_waypoint_fields(
         DBSession.query(Waypoint).
+        options(joinedload(getattr(Waypoint, 'geometry'))).
```

Comment:
Do you mean something like 152f98b ?

For the WP we obviously only need the ``geom``. For associated routes (not managed yet), having the default geom is not relevant, if ``geom_detail`` is not available we'd rather not display the route on the map at all. If it is available that could be cool to display the route geom in the WP map. I just wonder what will happen if we have too many route geometries: will it make the map hard to read? will it make the page too heavy?